### PR TITLE
Add more props

### PR DESCRIPTION
**Pull request by FrederikBolding**

### CODE REVIEW

File: src/atoms/Badge.tsx [hunk 1]
```diff
@@ -10,11 +11,14 @@ export interface BadgeProps {
...
-export const Badge: FunctionComponent<BadgeProps> = ({ type, children }) => {
+export const Badge: FunctionComponent<BadgeProps & BoxProps> = ({ type, children, ...props }) => {
   const theme = useTheme();
 
   return (
-    <Box backgroundColor={theme.variants.banner[type].color} sx={{ borderRadius: 'banner' }}>
+    <Box
+      backgroundColor={theme.variants.banner[type].color}
+      sx={{ borderRadius: 'banner' }}
+      {...props}>
```
Comment (GuillaumeRx) [inline src/atoms/Badge.tsx:21]: the passed through `Badge` `sx` prop will not be taken into account

File: src/molecules/Banner.tsx [hunk 2]
```diff
@@ -23,20 +24,25 @@ const icons: { [key in BadgeType]: IconType } = {
...
   const [isExtended, setExtended] = useState(extended);
 
   const handleToggle = () => setExtended((value) => !value);
 
   return (
-    <Box mb="2" variant={`banner.${type}`} sx={{ borderRadius: 'banner' }}>
+    <Box
+      mb="2"
+      sx={{ borderRadius: 'banner' }}
+      {...getMarginProps(props)}
```
Comment (GuillaumeRx) [inline src/molecules/Banner.tsx:44]: same here

**Review by GuillaumeRx — CHANGES_REQUESTED** [review]

File: src/atoms/Badge.tsx (hunk 1 above)
Comment (FrederikBolding) [reply src/atoms/Badge.tsx:21]: See Slack 😄

**Review by GuillaumeRx — APPROVED** [review]
lgtm

**Pull request merged**